frontend-react/src/utils/ScreenWidth.ts

```diff
@@ -0,0 +1,11 @@
+export default function isScreenWidth(size: "mobile" | "tablet" | "desktop") {
```

Comment:
Minor point, but it might be nice to add convenience functions like `isMobile`, `isTablet`, `isDesktop` so we don't have to always know what to pass into this.  This function can still be the thing we use under the hood, though:

```js
export function isMobile() {
  return isScreenWidth("mobile");
}

export function isTablet() {
  return isScreenWidth("tablet");
}

...
```

Also, I think if this is going to be a general utility file for screen width operations, it may not make sense to export a single function as a default.

Reply:
This totally makes sense -- shouldn't have to rely on the developer to know the potential values, should just export them as semantic functions.